### ISSUE TITLE
npm ERR! node-sass@4.9.0 install: `node scripts/install.js`

### ISSUE THREAD

**Issue by sonicviz** [wprig/wprig#294]
## Issue Overview

Running npm install fails due to:

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-sass@4.9.0 install: `node scripts/install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@4.9.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/devilbox/.npm/_logs/2019-01-23T04_03_30_733Z-debug.log

## Describe your environment

Win 10 x64 pro

Running Docker with Devilbox https://devilbox.readthedocs.io/en/latest/ 

Using a PHP 7.2 container with the following tools and versions installed:

composer	1.8.0
drupal-console	1.8.0
drush7	7.4.0
drush8	8.1.18
drush9	9.5.2
git	2.18.0
Laravel installer	2.0.1
mysqldump-secure	0.16.3
node	8.15.0
npm	6.4.1
Phalcon devtools	3.4.0
Symfony installer	1.5.11
Wordpress cli	2.1.0

devilbox@php-7.2.14 in /shared/httpd $ gulp -v
[04:58:32] CLI version 2.0.1
 

## Steps to Reproduce

1. Set up a clean install of Devilbox on Win 10
2. Install a fresh version of WordPress and update
3. Install the wprig master directory into themes 
4. Open the Devilbox PHP container with shell.bat , navigate to the wprig theme directory and run "npm install"

## Expected Behavior

Successful completion of npm install

## Current Behavior

Failed on error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-sass@4.9.0 install: `node scripts/install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@4.9.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/devilbox/.npm/_logs/2019-01-23T04_03_30_733Z-debug.log

## Possible Solution

Probably a tool or package version conflict somethere, going on past experience with Meteor/Vue but just guessing.

**Comment by mor10** [comment]
What version of WP Rig are you running?

**Comment by sonicviz** [comment]
Latest downloaded from github

**Comment by ehowey** [comment]
I had this same issue.  Fixed it by installing node.js v.10.x.  I was using Homebrew on Mac OS...when I ran "brew install node" it installed node.js v.11.x which was triggering some kind of script error on npm install.  I am new to terminal but I hope that helps...

**Comment by sonicviz** [comment]
I shouldn't have to revert to a previous version of node to get it running though.

**Comment by mor10** [comment]
I'm not able to reproduce running latest version of Node. @sonicviz, can you try again using a new folder? Also, try using the `v2.0` branch to see if the problem persists across both versions.

**Comment by ehowey** [comment]
I tracked down what is causing this when I was switching between v.104 and v.20 of WP-Rig.  In v1.04 it includes node-sass 4.9 which according to their documentation (https://github.com/sass/node-sass) is not compatible with Node.js v.11.xx, only Node.js v.10.xx.  This discrepancy causes the error.  I believe if sonicviz upgraded to Node.js v.10.xx this would be fixed.  I had to downgrade from v.11.xx whereas they would have to upgrade...
